subprojects/platform-jvm/src/test/groovy/org/gradle/jvm/toolchain/internal/ShowToolchainsTaskTest.groovy

```diff
@@ -278,6 +276,10 @@ $errorLines
         providerFactory
     }
 
+    private static InstallationLocation testLocation(File javaHome) {
```

Comment:
I would say pass in a String to this method and then call 'new File' only here, not at every usage site. Ie. I see how this is an automatic refactoring, but let's improve it.

Reply:
Done, thanks!